src/Helpers/HelperTrait.php

```diff
@@ -2,8 +2,27 @@
 
 namespace GingerPluginSdk\Helpers;
 
+use GingerPluginSdk\Collections\AbstractCollection;
+
 trait HelperTrait
 {
+    public static function isSameType($argument1, $argument2): bool
+    {
+        if (is_object($argument1)) {
+            if (is_object($argument2)) {
```

Comment:
Could we use `if(is_object($argument1) && is_object($argument2))`?

Reply:
Okay will updates this moment